src/js/handlebars.helpers.js

```diff
@@ -102,6 +102,63 @@ export default function handlebarsHelpers(handlebars) {
     // Call the formatDate helper with the determined date and format
     return handlebars.helpers.formatDate(dateString, dateToFormat, format);
   });
+
+  /**
+   * Format time duration into a string.
+   *
+   * Supports two formats: "short" and "long".
+   * - Short format: Display the duration in simplified format of "HH:MM:SS". It is the default format.
+   * - Long format: Display the duration in descriptive format of "X hours Y minutes Z seconds".
+   *
+   * @param {Object} duration - Duration object with properties 'hours', 'minutes', and 'seconds'.
+   * @param {String} format - Option for format type 'short' or 'long'. If none provided, 'short' is the defaut value.
+   *
+   * @returns {String} Formatted duration
+   * Examples:
+   * - 03:00:00 - 3 hours
+   * - 03:15:00 - 3 hours 15 minutes
+   * - 01:30:45 - 1 hour 30 minutes 45 seconds
+   * - 02:00:45 - 2 hours 45 seconds
+   * - 07:12 - 7 minutes 12 seconds
+   * - 00:45 - 45 seconds
+   *
+   * Usage:
+   * {{formatDuration duration}}
+   * {{formatDuration duration long}}
+   */
+  handlebars.registerHelper('formatDuration', function(duration, format) {
```

Comment:
is this going to take an object or a string if you did use the split? unsure if your sample data is what you are expecting for this helper.

Reply:
Same answer as previous comment.